form-guidelines.rst

```diff
@@ -0,0 +1,186 @@
+******************
+Form Guidelines
+******************
+
+.. note::
+
+  This page is a reference guide for users who edit the raw XML in ODK forms. If you use the XLSForm form designer, see http://xlsform.org for form design help.
```

Comment:
I don't know if intro content, which sets the context for understanding the whole doc, belongs in a note.